encorecloud/pubsub.go

```diff
@@ -24,12 +24,18 @@ const (
 
 // PublishToTopic publishes the specified attrs and data to the topic specified by topicID.
 //
+// orderingKey should be empty on topics not configured with an OrderingKey, otherwise it
+// should be given a string value. It will be used to send messages to FIFO queues
```

Comment:
Nit: present tense over future tense
```suggestion
// should be given a string value. It is used to send messages to FIFO queues
```